src/table/utils/styling-utils.ts

```diff
@@ -15,7 +15,6 @@ export const STYLING_DEFAULTS = {
   EXCLUDED_BACKGROUND:
     'repeating-linear-gradient(-45deg, rgba(200,200,200,0.08), rgba(200,200,200,0.08) 2px, rgba(200,200,200,0.3) 2.5px, rgba(200,200,200,0.08) 3px, rgba(200,200,200,0.08) 5px)',
   WHITE: '#fff',
-  PADDING: '7px 14px',
```

Comment:
As we already had the default padding in the muiTheme https://github.com/qlik-oss/sn-table/blob/main/src/table/mui-setup.js#L13